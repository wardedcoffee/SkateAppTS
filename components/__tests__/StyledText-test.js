import * as React from 'react';
import renderer from 'react-test-renderer';

import { MonoText } from '../StyledText';

it(`renders correctly`, () => {
  const tree = renderer.create(<MonoText>Snapshot test!</MonoText>, {
    createNodeMock: (element) => {
      return document.createElement(element.type);
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});