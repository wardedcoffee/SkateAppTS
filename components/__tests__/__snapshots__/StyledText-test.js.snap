// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      [
        undefined,
        {
          "fontFamily": "quicksand-bold",
        },
      ],
    ]
  }
>
  Snapshot test!
</Text>
`;
